MoltenVK/MoltenVK/Utility/MVKBaseObject.h

```diff
@@ -224,3 +224,16 @@ class MVKConfigurableMixin {
 static inline const MVKConfiguration& mvkGetMVKConfig(MVKBaseObject* mvkObj) {
 	return mvkObj ? mvkObj->getMVKConfig() : getGlobalMVKConfig();
 }
+
+/**
+ * Returns the reporting level string associated with the specified MoltenVK log level.
+ */
+static inline const char* mvkGetReportingLevelString(MVKConfigLogLevel logLevel) {
```

Comment:
```suggestion
/** Returns the reporting level string associated with the specified MoltenVK log level. */
static constexpr char* mvkGetReportingLevelString(MVKConfigLogLevel logLevel) {
```

Reply:
Causes warnings for static library builds (on Ventura, Xcode 15.2):

`ISO C++11 does not allow conversion from string literal to 'char *' `